### ISSUE TITLE
Text length calculation seems wrong

### ISSUE THREAD

**Issue by lobau** [Volumetrics-io/mrjs#487]
I feel like this problem keeps popping.
If you look at this screenshot, I would expect all 3 lines to be stacked on top of each other, without line breaks in the middle:

![image](https://github.com/Volumetrics-io/mrjs/assets/6750584/36565e2c-fa6b-42f5-ab94-b4a5708aa418)

https://docs.mrjs.io/doc/mr-a/#example

I believe the line break is because we miscalculate the width of the text.

**Comment by bkcastro** [comment]
It looks like its also affects the mr-div page 
![mr-div example page](https://github.com/Volumetrics-io/mrjs/assets/66087694/1060a21b-c5b8-4aee-b008-27bf2c8e3180)

**Comment by bkcastro** [comment]
Might be a problem with mr div entity, ill try to poke around

**Comment by bkcastro** [comment]
I did some poking around and found that the width calculated is correct but for some reason, the last word gets sent to a new line. Sample image below. An easy fix is to just give the textObjs a bigger maxWidth but this makes the bounding box clip with the right side of the text. I'm not too sure how this may interact with different CSS properties.
![Screenshot 2024-03-13 132927](https://github.com/Volumetrics-io/mrjs/assets/66087694/aec3628e-ff7e-4217-9edd-1c1330bc02a7)
Here is the clipping example: 
![mrjsTextSimpleFix](https://github.com/Volumetrics-io/mrjs/assets/66087694/c7ba9195-28e2-4e74-a5c7-39ef044a2cb7)

**Comment by bkcastro** [comment]
@hanbollar Yeah I plan on fixing the text issue later today just have some school work to get done first. Thanks for clearing things up.
